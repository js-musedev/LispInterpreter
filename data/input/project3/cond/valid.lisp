(COND (NIL 54) (NIL 34) (34 78) (645 234))