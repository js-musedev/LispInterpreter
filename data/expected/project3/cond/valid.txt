78
23
2
12